src/main/deploy/shooterData.csv

```diff
@@ -1,10 +1,17 @@
 Distance (inches),Angle (degrees),Velocity (RPM)
 Actual point(s)
-60,25,2000
-81,32,2000
-110,38,2100
-142,42,2200
-183,50,2600
-220,53,3000
-300,58,3700
-332,60,4200
\ No newline at end of file
+0, 7.3, 2700
+
+
+
+80,15.77,2375
+100,17.41,2375
+121,23.75,2416
+141,24.37,2518.77
+162, 30.31, 2539
+181, 32.97, 2662.12
+200, 34.61, 2700
+240, 37.88, 2887
+280, 40.34, 3215
+320, 47.71, 3500
+360, 49.76, 3911
```

Comment:
We should consider adding 1000000, 49.76, 3911 to smooth any unexpectedly large distance data points.
Might help with hood unexpectedly jumping way to high at times.